### ISSUE TITLE
torrent trackers blocked 

### ISSUE THREAD

**Issue by chirag127** [badmojr/1Hosts#210]
bt4.rutracker.cc
router.bittorrent.com
please remove these domains from the 1hosts pro.

**Comment by badmojr** [comment]
@jawz101 : bt4.rutracker.cc from https://raw.githubusercontent.com/AdAway/adaway.github.io/master/hosts.txt .

**Comment by chirag127** [comment]
Yeah it seems like because it is in https://raw.githubusercontent.com/AdAway/adaway.github.io/master/hosts.txt , it is present in most of the lists.

**Comment by badmojr** [comment]
Pinged it's dev. Be fixed on the upcoming update.

**Comment by chirag127** [comment]
What about router.bittorrent.com ?

**Comment by badmojr** [comment]
> router.bittorrent.com

From:
https://github.com/scomper/surge-list/blob/master/reject.list
https://github.com/lhie1/Rules/blob/master/Auto/REJECT.conf

**Comment by jawz101** [comment]
removed.  thanks
